content/src/content/jcr_root/apps/core/wcm/components/cloudviewer/v1/cloudviewer/cloudviewer.html

```diff
@@ -0,0 +1,53 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.cloudViewerModel="com.adobe.cq.wcm.core.components.models.CloudViewer"/>
+
+<style>
```

Comment:
CSS and Javascript should be added to the page just once, not with every instance of the component.

Reply:
Vendor JS should be added only once, it's not specific to the given instance of the component and can result in multiple inclusions when having more than one of these components on the same page:

```
<script src="https://documentcloud.adobe.com/view-sdk/main.js"></script>
```

Reply:
Fixed in 57dcdf2